iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/pointer_repository.inl

```diff
@@ -40,10 +40,11 @@ inline bool PointerRepository<id_t, ptr_t, CAPACITY>::registerPtr(id_t id, ptr_t
     if (m_info[id].basePtr == nullptr)
     {
         m_info[id].basePtr = ptr;
-        // AXIVION Next Construct AutosarC++19_03-A5.2.4 : Cast is needed for pointer arithmetic and casted back to the
-        // original type
-        // NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
-        m_info[id].endPtr = reinterpret_cast<ptr_t>(reinterpret_cast<uintptr_t>(ptr) + size - 1U);
+        m_info[id].endPtr =
+            // AXIVION Next Construct AutosarC++19_03-A5.2.4 : Cast is needed for pointer arithmetic and casted back to
+            // the original type
+            // NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
+            reinterpret_cast<ptr_t>(reinterpret_cast<uintptr_t>(ptr) + size - 1U);
```

Comment:
🤷

Reply:
Could you move the comment again above the line with reinterpret_cast and format the file? You have to keep in mind that some older devs might already have forgotten what happened before the comment when there are so many commented lines :sweat_smile:

Reply:
@elBoberido I suppose that's not possible, because when using "next line" both clang-tidy and Axivion really only suppress the next line and that wouldn't work in that case.

Reply:
@elBoberido Ok got it now. Fixed!